doc/usage.rst

```diff
@@ -532,3 +532,16 @@ no difference.
 
 Note that ``--prefix`` and ``--libdir`` are only supported if a single build
 context is in use.
+
+Relocation Mode
+---------------
+
+The installation can be done in specific mode (``--relocation``) for creating a
+directory that can be moved around. In that case the executables installed will
+lookup the sites (cf :ref:`sites`) of the packages relatively to its location.
```

Comment:
I think it's hard to figure out what `its location` is referring to here. Should it say `the location of the executable`?